test/evergreen.yml

```diff
@@ -40,6 +40,25 @@ functions:
       params:
         path: "WT_TEST-big-endian.tgz"
         destination: "wiredtiger/build_posix/test/format"
+  "fetch artifacts from many-coll" :
+    - command: s3.get
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        remote_file: dbpath_100k_500.tgz
+        bucket: build_external
+        local_file: dbpath_100k_500.tgz
+    - command: shell.exec
+      params:
+        shell: bash
+        script: |
+          set -o errexit
+          set -o verbose
+          mkdir -p mongo-tests/largescale/many-collection/dbpath
```

Comment:
Is it possible to extract the tarball into an existing directory path, to save this path creation command?

Reply:
If we extract the tarball into an existing path, I will still have to move everything to `mongo-tests/largescale/many-collection/dbpath` at a later stage

Reply:
Fair enough, I guess the folder path created is where the script expects the artifacts files locate.